src/code/z_camera.c

```diff
@@ -6014,7 +6017,7 @@ s32 Camera_Demo5(Camera* camera) {
     }
 
     sDemo5PrevAction12Frame = camera->globalCtx->state.frames;
-    Camera_ChangeSettingFlags(camera, CAM_SET_DEMOC, (4 | 1));
+    Camera_ChangeSettingFlags(camera, CAM_SET_CS_C, (4 | 1));
```

Comment:
I never actually determined if `C` was suppose to be `12` or an abbreviation for something. I always assumed it was abbreviated for something else like cutscene, similar to how there's `Door0` and `DoorC`, In my mind it always stood for `Cutscene` but I've yet to look closely at the cutscene settings. It's probably not `Cutscene` since they use the word `Demo`, and maybe it is just 12. This is the most commonly used setting for OnePointCutscenes though.